packages/react-components/src/components/InviteAgents/InviteAgents.tsx

```diff
@@ -0,0 +1,177 @@
+    const additionalAgentsNumber = availableAgentsNumber - visibleAgents.length;
+    const hasOnlyUnavailableAgents =
+      agents.length > 0 && availableAgentsNumber === 0;
+
+    return {
+      availableAgentsNumber,
+      visibleAgents,
+      additionalAgentsNumber,
+      hasOnlyUnavailableAgents,
+    };
+  }, [agents]);
+
+  const menuOptions = [
+    {
+      key: 'chatbot',
+      onClick: onSetUpChatbotClick,
+      element: (
+        <ActionMenuItem leftNode={<Icon source={ChatBotColored} />}>
+          Set up ChatBot
+        </ActionMenuItem>
+      ),
+    },
+    {
+      key: 'teammate',
+      element: (
+        <ActionMenuItem leftNode={<Icon source={PersonAdd} />}>
+          Invite teammate
+        </ActionMenuItem>
+      ),
+      onClick: onAddTeammateClick,
+    },
+  ];
+
+  return (
+    <div
+      className={cx(
+        ThemeClassName.Dark,
+        styles[baseClass],
+        {
+          [styles[`${baseClass}--empty`]]: agents.length === 0,
+          [styles[`${baseClass}--only-unavailable`]]: hasOnlyUnavailableAgents,
+        },
+        className
+      )}
+    >
+      {agents.length > 0 && (
+        <Tooltip
+          isVisible={isMenuOpen ? false : undefined}
+          offsetMainAxis={tooltipArrowOffset}
+          floatingStrategy="fixed"
+          hoverOnDelay={50}
+          className={cx(ThemeClassName.Dark, styles[`${baseClass}__tooltip`])}
+          triggerRenderer={
+            hasOnlyUnavailableAgents ? (
+              <div className={styles[`${baseClass}__not-accepting`]}>
+                <div
+                  className={cx(
+                    styles[`${baseClass}__not-accepting-status-dot`]
+                  )}
+                />
+                <Text noMargin bold size="sm">
+                  No active agents
+                </Text>
+              </div>
+            ) : (
+              <div className={styles[`${baseClass}__tooltip-trigger`]}>
+                <div className={styles[`${baseClass}__avatar-container`]}>
+                  {visibleAgents.map((agent, index) => (
+                    <Avatar
+                      className={styles[`${baseClass}__avatar`]}
+                      key={agent.email}
+                      type="image"
+                      size="xsmall"
+                      status={agent.status}
+                      src={agent.avatar}
+                      style={{ zIndex: agents.length - index }}
+                    />
+                  ))}
+                </div>
+                {additionalAgentsNumber > 0 && (
+                  <Text
+                    className={styles[`${baseClass}__available-agents-number`]}
+                  >
+                    +{additionalAgentsNumber}
+                  </Text>
+                )}
+              </div>
+            )
+          }
+        >
+          {hasOnlyUnavailableAgents ? (
+            <Interactive
+              header="No one's available to assist customers"
```

Comment:
IMO a request to change copy here or elsewhere will appear sooner rather than later. 😅 Shouldn't we leave this open for custom string? 🤔

Reply:
There are at least a few different places with this copy, so if we want to move it to props then other places should be covered too, so I'd leave it like that for now